.github/workflows/ci.yml

```diff
@@ -665,12 +665,21 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: ${{ vars.TRINO_AWS_REGION }}
-          S3_BUCKET: ${{ vars.TRINO_S3_S3FS_TESTS_BUCKET }}
         if: >-
           contains(matrix.modules, 'trino-filesystem-s3') && contains(matrix.profile, 'cloud-tests') &&
           (env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '')
         run: |
+          source .github/bin/s3/setup-empty-s3-bucket.sh
```

Comment:
source will leak script local variables, eg S3_SCRIPTS_DIR

run it as a normal script and the cat `.bucket-identifier` to initialize EMPTY_S3_BUCKET

Reply:
I did something similar to this, but also added `EMPTY_S3_BUCKET` as a github environment variable